syft/serde/torch_serde.py

```diff
@@ -140,7 +200,15 @@ def _simplify_torch_tensor(worker: AbstractWorker, tensor: torch.Tensor) -> bin:
     tags = tensor.tags
     if tags is not None:
         tags = list(tags)
-    return (tensor.id, tensor_bin, chain, grad_chain, tags, tensor.description)
+    return (
+        tensor.id,
+        tensor_bin,
+        chain,
+        grad_chain,
+        tags,
+        tensor.description,
+        syft.serde._simplify(worker, worker.serializer),
```

Comment:
Don't need to simplify it's suppoed to be a string

Reply:
Well strings are also simplified to `(5, (b'blah',))`

Reply:
Agreed